indico/htdocs/js/indico/RegistrationForm/registrationform.js

```diff
@@ -205,11 +209,14 @@ ndRegForm.directive('ndRegForm', function($rootScope, url, sortableoptions, regF
                 removeSection: function(section) {
                     regFormFactory.Sections.remove({
                         confId: $rootScope.confId,
-                        sectionId: section.id
-                    }, {}, function(updatedSections) {
-                        regFormFactory.processResponse(updatedSections, {
-                            success: function(updatedSections)  {
-                                $scope.sections = updatedSections;
+                        sectionId: section.id,
+                        confFormId: $rootScope.confFormId
+                    }, {}, function(data) {
+                        regFormFactory.processResponse(data, {
+                            success: function()  {
+                                $scope.sections = $scope.sections.filter(function(obj) {
```

Comment:
Personally, I don't like the idea of returning all the data related to sections from the controller. And that is the reason why I changed it. Is there any valid reason why I should do it the way it had been made before?